pkg/tsdb/tsdbtest/config_test.go

```diff
@@ -29,6 +29,9 @@ func TestGetV3ioConfigPath(t *testing.T) {
 		{description: "get config from package testdata",
 			expectedPath: filepath.Join(TsdbDefaultTestConfigPath, config.DefaultConfigurationFileName),
 			setup: func() func() {
+				// Make this test agnostic to environment variables at runtime (store & recover on exit)
```

Comment:
This is a unit test. Unit tests don't really use the config since.. well.. they are unit tests so this change might be redundant.